### PR TITLE
Fix rails 61 issue

### LINKED ISSUE #197: Rails 6.1 - undefined method `delegate' for ERBLint::Reporter:Class
Just upgraded to Rails 6.1 and am now getting this error:

```
/Users/jko/.asdf/installs/ruby/2.7.2/lib/ruby/gems/2.7.0/bundler/gems/erb-lint-05596ea189ee/lib/erb_lint/reporter.rb:36:in `<class:Reporter>': undefined method `delegate' for ERBLint::Reporter:Class (NoMethodError)
```

Is `delegation` [now included](https://github.com/rails/rails/blob/6-1-stable/activesupport/lib/active_support/core_ext/module.rb#L10) from `module.rb`?

### PR DESCRIPTION
**Pull request by jhottenstein**
Perhaps the `bundle update` is a little extreme but it exposes #197 in the test suite.

Is there a reason to activesupport in the Gemfile and the gemspec?

Closes #197

### CODE REVIEW

File: lib/erb_lint/reporter.rb [hunk 1]
```diff
@@ -1,5 +1,6 @@
 # frozen_string_literal: true
 require 'active_support/core_ext/class'
+require 'active_support/core_ext/module/delegation'
```
Comment (exterm) [inline lib/erb_lint/reporter.rb:3]: Will this break compatibility with earlier active_support versions?

Comment (exterm) [reply lib/erb_lint/reporter.rb:3]: Checked locally and it looks like it doesn't.

**Review by exterm — APPROVED** [review]
Thank you for the fix! Tested this against our rails-master monolith and this works. Should have included this in the release 🤦

**Comment by exterm** [comment]
I'm OK with the full `bundle update`. Makes sense to test against up-to-date dependencies anyway.

**Comment by kwent** [comment]
Seeing the same issue. Can we cut a new release ? Regards

**Comment by exterm** [comment]
@jhottenstein if you rebase I'll merge and cut a new release.

**Comment by rickychilcott** [comment]
Thank you for merging and cutting a new gem @exterm and for the contributors who helped us get there (@jhottenstein and @siccous)!

**Comment by kwent** [comment]
Thank you all! 😙

**Pull request merged**